hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -227,27 +227,31 @@ private static void populateInvalidTableConfigFilePathProps(TypedProperties prop
     props.setProperty("hoodie.deltastreamer.ingestion.uber_db.dummy_table_uber.configFile", dfsBasePath + "/config/invalid_uber_config.properties");
   }
 
-  private static void populateCommonProps(TypedProperties props) {
+  protected static void populateCommonProps(TypedProperties props, boolean setKafkaConfig, boolean setHiveConfig) {
```

Comment:
Can we split the settings works for Kafka and Hive into different methods so that we can make the method smaller and have more scalability?